app/grandchallenge/github/tasks.py

```diff
@@ -51,78 +52,98 @@ def get_repo_url(payload):
     return repo_url.replace("//", f"//x-access-token:{access_token}@")
 
 
-@shared_task()
+@shared_task(**settings.CELERY_TASK_DECORATOR_KWARGS["acks-late-2xlarge"])
 def get_zipfile(*, pk):  # noqa C901
     GitHubWebhookMessage = apps.get_model(  # noqa: N806
         app_label="github", model_name="GitHubWebhookMessage"
     )
     ghwm = GitHubWebhookMessage.objects.get(pk=pk)
-    payload = ghwm.payload
-    repo_url = get_repo_url(payload)
-    zip_name = f"{ghwm.repo_name}-{ghwm.tag}.zip"
-    tmp_zip = tempfile.NamedTemporaryFile()
-    has_open_source_license = False
-    license = "No license file found"
-    try:
-        recurse_submodules = Algorithm.objects.get(
-            repo_name=ghwm.payload["repository"]["full_name"]
-        ).recurse_submodules
 
-    except Algorithm.DoesNotExist:
-        recurse_submodules = False
-    with tempfile.TemporaryDirectory() as tmpdirname:
-        # Run git lfs install here, doing it in the dockerfile does not seem
-        # to work
+    if ghwm.zip_file_status != ZipStatusChoices.COMPLETED:
```

Comment:
I think this code should not run if the status is `STARTED` as well

Reply:
True, thanks for catching this!